doc/commands.md

```diff
@@ -128,48 +128,50 @@ Rewriting rules for definable symbols are declared using the `rule` command.
 ```
 rule add zero      $n ↪ $n
 rule add (succ $n) $m ↪ succ (add $n $m)
+rule mul zero      _  ↪ zero
 ```
 
-Note that rewriting rules can also be defined simultaneously, using the `with`
-keyword instead of the `rule` keyword for all but the first rule.
+Terms prefixed by the sigil `$` and `_` are pattern variables.
```

Comment:
What is "sigil"?

Reply:
Ah, it's symbols used to mark variables in general such as $ in shell, or @ for arrays in Perl &c. <https://en.wikipedia.org/wiki/Sigil_(computer_programming)>